### PR TITLE
added profile for balanced02

### PR DESCRIPTION
**Pull request by Balanced02**
=> Added profile for Balanced02
=> Added .env.example file so subsequent contributors wouldn't go through the stress of thinking about the env variables
=> Assign ids to obj using index to avoid over-writing datas with same ids

### CODE REVIEW

File: src/data/users/balanced02.json [hunk 1]
```diff
@@ -0,0 +1 @@
...alanced02","name":"Adepoju Daniel","location":"Abuja, Nigeria","avatar":"https://pbs.twimg.com/profile_images/1272287309615030272/qXE_sg-__400x400.jpg","currentRole":"Software Engineer","currentCompany":"Interswitch","portfolio":"https://github.com/Balanced02","bio":"Madly in love with JavaScript","category":"developer","social":{"twitter":"Balanced02","linkedIn":"balanced02","github":"Balanced02","email":"adebalanced02@gmail.com"},"skills":["React","React Native","Javascript","NodeJs","MongoDB"],"selectedWorks":[{"projectTitle":"Edubase","projectImageUrl":"https://edubaseforschools.com/wp-content/uploads/2020/05/stud_class.png","projectDescription":"A cloud based Virtual Learning Environment. Customized and branded for your school with special features which includes Classroom Management, Content, Resources, Result Management and lots more","projectLink":"https://www.edubase.app"},{"projectTitle":"VCF Tel Extractor","projectImageUrl":"https://upload.wikimedia.org/wikipedia/commons/thumb/d/db/Npm-logo.svg/1200px-Npm-logo.svg.png","projectDescription":"Openn source module to extract phone numbers from .vcf file","projectLink":"https://www.npmjs.com/package/vcftelextractor"}],"id":6}
```
Comment (Kunoacc) [inline src/data/users/balanced02.json:1]: you're not supposed to add the ID yourself. The generator adds it when you make changes

File: src/data/users/promise.json [hunk 2]
```diff
@@ -29,5 +29,6 @@
       "projectDescription": "Developed with React and Laravel",
       "projectLink": "https://quickrace.com.ng"
     }
-  ]
+  ],
+  "id": 5
```
Comment (Kunoacc) [inline src/data/users/promise.json]: here too

File: src/data/users/balanced02.json (hunk 1 above)
Comment (Balanced02) [reply src/data/users/balanced02.json:1]: The generator doesn't add it, This is supposed to be in my next PR, cause I noticed that and fixed it after I made this PR

File: src/data/users/promise.json (hunk 2 above)
Comment (Balanced02) [reply src/data/users/promise.json]: Gives undefined, thus, when two profiles are not set, one overrides the other, fixed this on my pc and would make a push for this to this pr instead

File: src/data/users/balanced02.json (hunk 1 above)
Comment (Balanced02) [reply src/data/users/balanced02.json:1]: Kindly check the updated codes, I've made a push

Comment (Kunoacc) [reply src/data/users/balanced02.json:1]: Hmmm

Comment (Kunoacc) [reply src/data/users/balanced02.json:1]: Let me investigate this

Comment (samie820) [reply src/data/users/balanced02.json:1]: @Kunoacc Welldone, Sherlock Holmes...

**Pull request merged**